File: JS/URI/3302.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');


let caso = parseInt(lines.shift());

for(i=0; i < caso; i++)
    {
        resposta = parseInt(lines.shift());
        console.log("resposta "+ (i+1) +": "+resposta );
    }
    

File: C/URI/3302.c
#include<stdio.h>
int main(){

    int caso, i, problema;

    scanf("%d", &caso);

    for(i=1; i<=caso; i++)
    {
        scanf("%d", &problema);
        printf("resposta %d: %d\n", i, problema);
    }


    return 0;
}